dnsimple/registrar_registrant.go

```diff
@@ -0,0 +1,161 @@
+package dnsimple
+
+import (
+	"context"
+	"fmt"
+)
+
+type CreateRegistrantChangeInput struct {
+	DomainId           int               `json:"domain_id"`
```

Comment:
I realized that this should be a `string` type as our domain id could be also the actual domain name.

Reply:
Fixed in [a0bab60](https://github.com/dnsimple/dnsimple-go/pull/146/commits/a0bab607f157c5a7a5600ef8036c3a8a7bbd4d4f)